services/mongoose.js

```diff
@@ -29,10 +29,10 @@ const enabled = require('debug').enabled;
 // Pull the mongo url out of the environment.
 let url = process.env.TALK_MONGO_URL;
 
-// Reset the mongo url in the event it hasn't been overrided and we are in a
-// testing environment. Every new mongo instance comes with a test database by
+// Reset the mongo url if we are in a testing environment.
+// Every new mongo instance comes with a test database by
 // default, this is consistent with common testing and use case practices.
-if (process.env.NODE_ENV === 'test' && !url) {
+if (process.env.NODE_ENV === 'test') {
```

Comment:
Development should **not** use the test database. When you start the application, you specify the database to use. When you run tests, you do **not load configuration into the environment**, as tests _would not be predictable if it depended on local configuration_. Hence, this change is not necessary, and counter to what is expected if you do override the url.